Symfony/CS/Tokenizer/Token.php

```diff
@@ -149,6 +145,25 @@ public function equalsAny(array $others, $caseSensitive = true)
     }
 
     /**
+     * A helper method used to find out whether or not a certain input token has to be case-sensitively matched.
+     *
+     * @param bool|bool[] $caseSensitive global case sensitiveness or an array of booleans, whose keys should match
+     *                                   the ones used in $others. If any is missing, the default case-sensitive
+     *                                   comparison is used.
+     * @param int|string  $key           the key of the token that has to be looked up
```

Comment:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/8bea2fe91fd6c77b7ca2af91b1bb4f81cd1c1e73/Symfony/CS/Tests/Tokenizer/TokenTest.php#L344

and 

https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/973/files#diff-355c70b81d3f4f9e8a5fe54b9d86d161R294

are testing string keys.